mmvb_backend/settings.py

```diff
@@ -156,3 +158,6 @@
 
 CELERY_BROKER_URL = "redis://localhost"
 CELERY_RESULT_BACKEND = "redis://"
+
+# CORS settings
+CORS_ORIGIN_ALLOW_ALL = True
```

Comment:
Maybe just add a `TODO` here to remind us to properly configure CORS in the future?